packages/zoe/test/swingsetTests/zoe/vat-alice.js

```diff
@@ -352,16 +349,18 @@ const build = async (E, log, zoe, issuers, payments, installations, timer) => {
       TokenA: moolaIssuer,
       TokenB: simoleanIssuer,
     });
-    const invite = await E(zoe).makeInstance(
+    const bogusInvite = await E(zoe).makeInstance(
```

Comment:
I think `bogus` implies fake, but this invite isn't fake, right? Maybe we should rename to `unused` or something like that.

Reply:
`unusedInvite` is good.

Reply:
Took your suggestion of having the initial invite be an invite to add liquidity. So `unusedInvite` is gone.